ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx

```diff
@@ -289,6 +293,54 @@ function FlowsTable({
                     </Tbody>
                 );
             })}
+            {numAllowedEgressFlows > 0 && (
+                <Tbody>
+                    <Tr>
+                        <Td />
+                        {isEditable && <Td />}
+                        <Td dataLabel={columnNames.entity}>
+                            <Flex direction={{ default: 'row' }}>
+                                <FlexItem>
+                                    <div>+ {numAllowedEgressFlows} allowed flows</div>
+                                    <div>
+                                        <TextContent>
+                                            <Text component={TextVariants.small}>
+                                                Across this cluster
+                                            </Text>
+                                        </TextContent>
+                                    </div>
+                                </FlexItem>
+                            </Flex>
+                        </Td>
+                        <Td dataLabel={columnNames.direction}>Egress</Td>
+                        <Td dataLabel={columnNames.portAndProtocol}>Any / Any</Td>
+                    </Tr>
+                </Tbody>
+            )}
+            {numAllowedIngressFlows > 0 && (
+                <Tbody>
+                    <Tr>
+                        <Td />
+                        {isEditable && <Td />}
+                        <Td dataLabel={columnNames.entity}>
+                            <Flex direction={{ default: 'row' }}>
+                                <FlexItem>
+                                    <div>+ {numAllowedIngressFlows} allowed flows</div>
+                                    <div>
+                                        <TextContent>
+                                            <Text component={TextVariants.small}>
+                                                Across this cluster
+                                            </Text>
+                                        </TextContent>
+                                    </div>
+                                </FlexItem>
+                            </Flex>
+                        </Td>
+                        <Td dataLabel={columnNames.direction}>Ingress</Td>
+                        <Td dataLabel={columnNames.portAndProtocol}>Any / Any</Td>
```

Comment:
fwiw we can probably factor this out in the future since it seems to be pretty much the same thing minus `ingress` -> `egress` and vise versa

Reply:
Sure we can do that. I made a reusable component